pydoctor/templates/apidocs.css

```
@@ -195,30 +202,90 @@ tr.private {
     margin-bottom: 6px;
 }
 
+/* This is magic to show the transparent index link */
+a.navbar-brand {
+    /* background-color:#f3d3db; */
+    border:solid;
+    margin-right: 0.3em;
+    border-width: 1.5px;
+    border-color: #c7254e;
+    background-color:#f9f2f4;
+    border-radius: 4px;
+}
+
+/* This is for the main top navagation bar */
+.page-header > h1 > code {
+    background-color:#f9f2f4;
+    /* background-color:transparent; */
+}
+
+/* .page-header > h1 > code > a {
+    background-color:#FCF9FA;
+} */
+
+/* 
+<a><code></code><a> for links
+This is inline docstring content marked up as code, for example C{foo} in epytext
+*/
+a > code {
```

```suggestion
a[href] > code {
```
The `<a>` tag is also used to define anchors and that should not cause the text to become red.

---

Where do we use this rule? I'd expect most HTML would have the `<a href="">` inside a `<code>` tag and not vice versa.

---

> Where do we use this rule?

Here for instance: file:///Users/tristan/Documents/pydoctor/build/apidocs/pydoctor.epydoc.markup.ParsedDocstring.html

The `Subclasses must implement L{to_stan()}.`

---

Hmm, we could change the code that outputs this in `pydoctor.epydoc.markup.*`, but I'm not sure whether that would make it easier or harder to write the CSS.